google-cloud-spanner/lib/google/cloud/spanner/session.rb

```diff
@@ -198,29 +190,10 @@ def delete_session
         #     puts "User #{row[:id]} is #{row[:name]}""
         #   end
         #
-        # @example Query without streaming results:
-        #   require "google/cloud/spanner"
-        #
-        #   spanner = Google::Cloud::Spanner.new
-        #
-        #   db = spanner.client "my-instance", "my-database"
-        #
-        #   results = db.execute "SELECT * FROM users WHERE id = @user_id",
-        #                        params: { user_id: 1 },
-        #                        streaming: false
-        #
-        #   user_row = results.rows.first
-        #   puts "User #{user_row[:id]} is #{user_row[:name]}""
-        #
-        def execute sql, params: nil, transaction: nil, streaming: true
+        def execute sql, params: nil, transaction: nil
           ensure_service!
-          if streaming
-            Results.execute service, path, sql,
-                            params: params, transaction: transaction
-          else
-            Results.from_grpc service.execute_sql \
```

Comment:
Can we remove `Service#execute_sql` and `Service#read` if they are no longer needed?